<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>

  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>

   <property name="styleSheet">
    <string notr="true">
QWidget#bgwidget[theme="light"] {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:1,
                                      stop:0 #DCC7AA,
                                      stop:1 #FFFFFF);
}

QWidget#bgwidget[theme="dark"] {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:1,
                                      stop:0 #0f172a,
                                      stop:1 #1e293b);
}

QWidget#bgwidget QLabel {
    font: 11pt "Segoe UI";
}

QWidget#bgwidget[theme="light"] QLabel {
    color: #3A2A20;
}

QWidget#bgwidget[theme="dark"] QLabel {
    color: #e2e8f0;
}

QWidget#bgwidget QLabel#label {
    font: 700 36pt "Segoe UI";
}

QWidget#bgwidget QLabel#label_2 {
    font: 16pt "Segoe UI";
}

QWidget#bgwidget QLabel#error {
    font: 10pt "Segoe UI";
    color: #FF4D4D;
}

QWidget#bgwidget QLineEdit {
    border-radius: 10px;
    padding: 6px 10px;
    font: 11pt "Segoe UI";
}

QWidget#bgwidget[theme="light"] QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #C97B5A;
    color: #3A2A20;
}

QWidget#bgwidget[theme="dark"] QLineEdit {
    background-color: #1e293b;
    border: 1px solid #3b82f6;
    color: #e2e8f0;
}

QWidget#bgwidget QLineEdit:focus {
    border: 1px solid #3b82f6;
}

QWidget#bgwidget QPushButton#signup {
    border-radius: 20px;
    font: 600 14pt "Segoe UI";
    padding: 8px 12px;
    border: none;
}

QWidget#bgwidget[theme="light"] QPushButton#signup {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #C97B5A,
                                      stop:1 #B06345);
    color: #FFFFFF;
}

QWidget#bgwidget[theme="light"] QPushButton#signup:hover {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #D18764,
                                      stop:1 #BD6D4F);
}

QWidget#bgwidget[theme="light"] QPushButton#signup:pressed {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #A3573C,
                                      stop:1 #8B4731);
}

QWidget#bgwidget[theme="dark"] QPushButton#signup {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #3b82f6,
                                      stop:1 #2563eb);
    color: #FFFFFF;
}

QWidget#bgwidget[theme="dark"] QPushButton#signup:hover {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #60a5fa,
                                      stop:1 #3b82f6);
}

QWidget#bgwidget[theme="dark"] QPushButton#signup:pressed {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #1d4ed8,
                                      stop:1 #1e40af);
}

QWidget#bgwidget QPushButton#backToLogin {
    background-color: transparent;
    border: none;
    border-radius: 0;
    font: 10pt "Segoe UI";
    padding: 0px;
    text-align: center;
}

QWidget#bgwidget[theme="light"] QPushButton#backToLogin {
    color: #C97B5A;
    text-decoration: underline;
}

QWidget#bgwidget[theme="dark"] QPushButton#backToLogin {
    color: #60a5fa;
    text-decoration: underline;
}

QWidget#bgwidget QPushButton#backToLogin:hover {
    opacity: 0.8;
}

/* Theme Toggle Button Styles */
QWidget#bgwidget QPushButton#themeToggle {
    background-color: transparent;
    border: none;
    border-radius: 25px;
}
QWidget#bgwidget QPushButton#themeToggle:hover {
    background-color: rgba(128, 128, 128, 0.2);
}
    </string>
   </property>

   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>300</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>

   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>380</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Register a brand new account</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>

   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeft</set>
    </property>
   </widget>

   <widget class="QLineEdit" name="emailfield">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>

   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeft</set>
    </property>
   </widget>

   <widget class="QLineEdit" name="passwordfield">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>

   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeft</set>
    </property>
   </widget>

   <widget class="QLineEdit" name="confirmpasswordfield">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Confirm password</string>
    </property>
   </widget>

   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>540</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>

   <widget class="QPushButton" name="signup">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>

   <widget class="QPushButton" name="backToLogin">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>630</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Already have an account? Log In</string>
    </property>
   </widget>


    <!-- THEME TOGGLE BUTTON -->
    <widget class="QPushButton" name="themeToggle">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>40</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>

    <!-- THEME TOGGLE BUTTON -->
    <widget class="QPushButton" name="themeToggle">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>40</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>

  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
